quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorGizmoKitchenSinkTest.java

```diff
@@ -72,7 +72,10 @@ void solve() throws ExecutionException, InterruptedException {
                 Collections.emptyList(),
                 HardSoftLongScore.ZERO);
 
-        SolverJob<TestDataKitchenSinkSolution, Long> solverJob = solverManager.solve(1L, problem);
+        SolverJob<TestDataKitchenSinkSolution, Long> solverJob = solverManager.solveBuilder()
+                .withProblemId(1L)
+                .withProblem(problem)
+                .run();
```

Comment:
Due to this PR, I realized this is used *a lot*. It seems to be *everywhere*. Maybe we un-deprecate this method on `SolverManager`?

Reply:
Agreed. Should we keep using the builder or revert changes for this method?

Reply:
I know it's extra work, but I'd prefer reverting this method back.